README.md

```
@@ -1 +1,176 @@
+- `make logs` - View container logs
+- `make clean` - Stop and remove containers/images
+- `make prod-deploy` - TODO: Deploy to GCP Cloud Run
+
+## Development
+
+### Local Development (without Docker)
+
+If you prefer to run the app locally without Docker:
+
+1. Install dependencies:
+```bash
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)